src/controller/CHIPDeviceController.cpp

```diff
@@ -82,6 +83,8 @@ constexpr uint16_t kMdnsPort = 5353;
 
 constexpr const uint32_t kSessionEstablishmentTimeout = 30 * kMillisecondPerSecond;
```

Comment:
nit for not your code: constexpr is already const, could clean up the above line.